.github/workflows/test-packer-qemu.yml

```diff
@@ -0,0 +1,49 @@
+name: Packer build qemu
+
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: codespaces
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+
+      - name: Setup packer
+        uses: hashicorp/setup-packer@1aa358be5cf73883762b302a3a03abd66e75b232 # v3.1.0
+        id: setup
+        with:
+          version: 1.11.2
+
+      - name: Generate cloud-init iso
+        run: |
+          sudo apt install genisoimage
+          echo '#cloud-config
+          chpasswd:
+            expire: False
+            users:
+            - {name: debian, password: password, type: text}
+          ssh_pwauth: True' > user-data
+          echo 'instance-id: iid-local01
+          local-hostname: localhost' > meta-data
+          genisoimage -output ci-data.iso -volid cidata -joliet -rock user-data meta-data
+      
+      - name: Install qemu
+        run: sudo apt-get update && sudo apt-get install qemu-system -y
+      
+      - name: Run packer init
+        id: init
+        run: packer init qemu.pkr.hcl
+
+      - name: Run packer build
+        id: build
+        run: packer build -var glueops_codespaces_container_tag=v0.62.0 qemu.pkr.hcl
```

Comment:
Can we have the container tag be a variable? or did we want to keep it hardcoded for now as we are testing it?

Reply:
This is the test file that will be deleted, it should be resolved in the original file